actnow/accounts/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import RegisterView
+
+urlpatterns = [
+    path("register/", RegisterView.as_view(), name="account_register"),
```

Comment:
```suggestion
    path("/", UserRegistrationView.as_view(), name="accounts-user_registration"),
```

Since we're doing RESTful, I take it posting to `/accounts` should be all one needs to do, right?